src/ServiceControl.RavenDB/EmbeddedDatabase.cs

```diff
@@ -146,7 +147,8 @@ public async Task<IDocumentStore> Connect(CancellationToken cancellationToken)
                 Conventions = new DocumentConventions
                 {
                     SaveEnumsAsIntegers = true
-                }
+                },
+                SkipCreatingDatabase = true
```

Comment:
I assume this is so we can create the DB(s) with Corax settings in the DatabaseSetup, correct?

Reply:
Yes, with this not being set before, we never actually hitting the path in `CreateDatabase` that would... create the database.